src/ml_board/frontend/Dashboard/src/components/settings/Settings.tsx

```diff
@@ -15,14 +15,18 @@ export const defaultSocketConnectionUrlHelperText = "eg: http://127.0.0.1:5002";
 export const defaultRestApiUrlHelperText = "eg: http://127.0.0.1:5001";
 
 export interface FuncProps {
-    validateConfigs(value:boolean): void;
+    setSocketConnectionRequest(): void;
     setConfigData(configData:settingConfigsInterface): void;
+    isConfigValidated?: boolean
 }
 
 // Settings file and Config popup file (except render part) might look exactly the same for now. But Settigns will have more functionality later. So we will add more things in Settings which will make it different from popup file.
 const Settings: React.FC<FuncProps> = (props) => {
 
     const isSocketConnected = useAppSelector(isConnected);
+    const grid_search_id = useAppSelector(getGridSearchId);
+    const rest_api_url = useAppSelector(getRestApiUrl);
+    const socket_connection_url = useAppSelector(getSocketConnectionUrl);
```

Comment:
something about the circulation of these 3 doesn't look good but they are working for now, so I accept them now but maybe we will change them in the future!

Reply:
Okay!